### ISSUE TITLE
Blank Advertising Summary

### ISSUE THREAD

**Issue by dhwlm** [facebookincubator/Facebook-For-OpenCart#606]
**FBE plugin version**
4.2.1

**OpenCart version**
OC 3.0.3.2

**Describe the bug**
I've installed the plugin, connect the assets/pages/ad accounts. Pixel seems to be firing, but the section Ads insight shows no data. However, The ads are running and I can see that through business dashboard.

**Screenshots**
![Screenshot 2022-01-12 at 3 31 15 PM](https://user-images.githubusercontent.com/19328150/149118950-2e39f526-a309-4221-bf76-78c4d1e42b3a.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome

**Additional context**
I'm not sure if this is a bug, or if i should be concerned about my integration. Any help would be appreciated.

**Comment by yao-tx** [comment]
Hi @dhwlm 

Do ensure that you are logged in to a Facebook Account that has access to your Facebook Business account. To do check if you are, simply open a new tab within the same browsing session and go to facebook.com or business.facebook.com.

**Comment by dhwlm** [comment]
Hi @yao-tx 

Yes, I did check this. I've tried clearing cache and resetting the connection. I tried to disable any adblock extensions, maybe that mightve caused this. I still see that section blank in opencart.

**Comment by yao-tx** [comment]
Hi @dhwlm 

When you click on the 'Manage Settings' button at the Facebook Business Extension page, does the pop up bring you to your Facebook Business page?

**Comment by dhwlm** [comment]
Hi @yao-tx ,

Yes it does, it shows connected to my opencart and also shows which FB page/Ad accounts/Catalog are connected to the integration.

**Comment by yao-tx** [comment]
Hi @dhwlm 

Did you have an older version of the Facebook Business Extension installed on your OpenCart website (specifically v3.1.2 and below) prior to installing the latest version (specifically v4.0.0 and above)?

**Comment by dhwlm** [comment]
@yao-tx 

Yes I did. Im not sure about the version number, but yes it was an old version that i was using. I recently upgraded to v4.2.1

**Comment by yao-tx** [comment]
Hi @dhwlm 

In that case, could you try to re-connect the Facebook Business Extension to your Facebook Business Account? To do so, please kindly refer to the following:

1) Login to your OpenCart Admin Dashboard, and on the left side menu, you should see the 'Facebook Business Extension' menu item. Click on the menu link.

2) Then, you should see a blue 'Uninstall' button, with the 'Manage Settings' button below it. Click on the 'Uninstall' button and a prompt should popup. Confirm and proceed. After it is completed, the extension should prompt you that it has successfully uninstalled, and the page should automatically refresh after you confirm the prompt. Otherwise, please try to perform a manual refresh.

3) Once that is done, you should see a 'Get started with Facebook' button instead of the two buttons in (2). Click on that and re-integrate your Facebook Business account with the extension.

4) Once everything is done, do refresh the Facebook Business Extension page and check if the ads summary section still remains blank.

**Comment by dhwlm** [comment]
Hi @yao-tx ,

I tried the above mentioned steps, I still don't see the Ads summary section. Its still blank (I have selected the Ads account from the dropdown).

Also, I'm not sure if this helps, I see some errors in the console on the extension page, attached screenshot.
<img width="384" alt="Screenshot 2022-01-21 at 3 20 17 PM" src="https://user-images.githubusercontent.com/19328150/150506553-74244134-2ff3-4be3-8e60-c56c92f54b1c.png">

**Comment by yao-tx** [comment]
Hi @dhwlm 

Could you open a support ticket with us on our [helpdesk](https://marketinsg.zendesk.com/)?

**Comment by dhwlm** [comment]
Thanks @yao-tx , I have created a support ticket.
